cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -102,9 +107,25 @@ type nodeTemplate struct {
 	Taints       []apiv1.Taint
 }
 
+var (
+	controlBurst *int
+	controlQPS   *float64
+	targetBurst  *int
+	targetQPS    *float64
+)
+
+func init() {
+	controlBurst = flag.Int("control-burst", rest.DefaultBurst, "Throttling burst configuration for the client to control cluster's apiserver.")
+	controlQPS = flag.Float64("control-qps", float64(rest.DefaultQPS), "Throttling QPS configuration for the client to control cluster's apiserver.")
+	targetBurst = flag.Int("target-burst", rest.DefaultBurst, "Throttling burst configuration for the client to target cluster's apiserver.")
+	targetQPS = flag.Float64("target-qps", float64(rest.DefaultQPS), "Throttling QPS configuration for the client to target cluster's apiserver.")
+}
```

Comment:
@amshuman-kr - As suggested, I have added rate-limiting parameters for both the APIServers as flags.